src/libcore/cmp.rs

```diff
@@ -481,6 +481,32 @@ pub trait Ord: Eq + PartialOrd<Self> {
     where Self: Sized {
         if self <= other { self } else { other }
     }
+
+    /// Returns max if self is greater than max, and min if self is less than min.
+    /// Otherwise this will return self.  Panics if min > max.
```

Comment:
I think this should follow our doc formatting conventions, no? I think we usually have a panics header for documenting panic'ing conditions.